src/tools/push_av_server/requirements.txt

```diff
@@ -1,5 +1,7 @@
 zeroconf
 cryptography
-uvicorn
 fastapi
 jinja2
+# We are currently patching hypercorn to add support for the TLS ASGI extension
+# As such updating its version should be linked to an update to the associated patch file
+Hypercorn==0.17.3
```

Comment:
`ModuleNotFoundError: No module named 'aioquic'` is seen when viewing the push_av_server UI, aioquic module can be included in the requirements. Always appreciated your efforts.